llvm/lib/Transforms/Utils/SCCPSolver.cpp

```diff
@@ -2125,6 +2143,21 @@ void SCCPInstVisitor::handleCallResult(CallBase &CB) {
   }
 }
 
+bool SCCPInstVisitor::isInstOverDefined(Instruction &Inst) {
```

Comment:
```suggestion
bool SCCPInstVisitor::isInstFullyOverDefined(Instruction &Inst) {
```
Maybe? To make it clearer that this only returns true if all the elements are overdefined.